app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -571,12 +556,21 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
         }
         return dialog;
     }
-    
+
     @Override
     public boolean isBackEnabled() {
-        return false;
+        return true;
     }
-    
+
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();

```

Comment:
Oh, you're totally right. 

I'm confused then, since this behavior is ~identical to the stock behavior when back is pressed, I think?